Services/Interfaces/web_interface/static/js/common/trackers.js

```diff
@@ -0,0 +1,32 @@
+
+$(document).ready(function() {
+
+    const getUserEmail = () => {
+        return $("#resources-urls").data("user-email")
+    }
+
+    const identify = (email) => {
+        posthog.identify(
+            'distinct_id',  // Replace 'distinct_id' with your user's unique identifier
+            { email: 'max@hedgehogmail.com', name: 'Max Hedgehog' } // optional: set additional person properties
+        );
+    }
+
+    const identifyIfNecessary = () => {
+        if (document.referrer.indexOf("/community_login") !== -1){
+            // come from login: identify
+            identify(getUserEmail());
+        }
+    }
+
+    const resetIfNecessary = () => {
+        if (getUserEmail() === ""){
+            // no logged in user and just logged out
+            // TODO: figure out when user just logged out
```

Comment:
this is a TODO: we don't want to all reset() everytime there is no email => how to know the user just logged out